geopandas/sindex.py

```diff
@@ -115,11 +355,12 @@ def intersection(self, coordinates, objects=False):
                 iter(coordinates)
             except TypeError:
                 # likely not an iterable
-                # this is a check that rtree does, we mimick it
+                # this is a check that rtree does, we mimic it
                 # to ensure a useful failure message
                 raise TypeError(
                     "Invalid coordinates, must be iterable in format "
-                    "(minx, miny, maxx, maxy) (for bounds) or (x, y) (for points)."
+                    "(minx, miny, maxx, maxy) (for bounds) or (x, y) (for points). "
+                    "Got `coordinates` = %s." % coordinates
```

Comment:
Can you use `format()` instead of `%` ?

Reply:
Of course!

I'm a big f-string user, but when I forget what Python X.Y supports, so I just default to `%` when in doubt. We are >=3.5 here so yeah `format` is much better.

Reply:
Fixed